pallets/liquidate/src/lib.rs

```diff
@@ -95,17 +95,16 @@ pub mod module {
 
     #[pallet::config]
     pub trait Config:
-        frame_system::Config
-        + CreateSignedTransaction<Call<Self>>
-        + pallet_loans::Config
-        + pallet_ocw_oracle::Config
+        frame_system::Config + CreateSignedTransaction<Call<Self>> + pallet_loans::Config
```

Comment:
Using `pallet_loans::Config` constraint may cause the same problem as this pr to solve. Maybe we can also add `LiquidateBorrow` trait later on.

Reply:
yes, later we can decouple this too